rust-sdk/whirlpool/src/token.rs

```diff
@@ -664,15 +664,34 @@ mod tests {
         assert_eq!(token_account.amount, amount);
     }
 
+    #[tokio::test]
+    #[serial]
+    async fn test_native_token_balance() -> Result<(), Box<dyn Error>> {
+        let ctx = RpcContext::new().await;
+        let amount = 1_000_000_000; // 1 SOL
+
+        // Setup native token (wSOL) account with balance using helper function
+        let wsol_ata = setup_ata_with_amount(&ctx, native_mint::ID, amount).await?;
```

Comment:
Is this testing a SDK function or a function from the test utils? In case the second I think we can skip this test